packages/perseus/src/widgets/definition/definition.test.ts

```diff
@@ -10,30 +15,22 @@ import {scorePerseusItemTesting} from "../../util/test-utils";
 import {renderQuestion} from "../__testutils__/renderQuestion";
 
 import type {PerseusDependenciesV2} from "../../types";
+import type {PerseusRenderer} from "@khanacademy/perseus-core";
 import type {UserEvent} from "@testing-library/user-event";
 
-const question = {
+const question: PerseusRenderer = generateTestPerseusRenderer({
     content:
         "Read the excerpt and answer the question below. \n\nThe Governor and Council of the Massachusetts had much conference many days; and at last . . . . concluded a peace and friendship with [[\u2603 definition 1]], upon these conditions.",
     images: {},
     widgets: {
-        "definition 1": {
-            graded: true,
-            version: {
-                major: 0,
-                minor: 0,
-            },
-            static: false,
-            type: "definition",
-            options: {
+        "definition 1": generateDefinitionWidget({
+            options: generateDefinitionOptions({
                 definition: "Definition text",
                 togglePrompt: "the Pequots",
-                static: false,
-            },
-            alignment: "default",
-        },
+            }),
+        }),
     },
-} as const;
```

Comment:
I'm guessing `as const` can be removed because we're using generators now? Or is it because of the typing? I see `as const` on one of the other variable definitions, but that one doesn't have a type

Reply:
Yeah, I think the `as const` can be removed since the types are correct now